datalad/__init__.py

```diff
@@ -16,6 +16,11 @@
 from .config import ConfigManager
 cfg = ConfigManager()
 
+from datalad.support.sshconnector import SSHManager
```

Comment:
not sure if I like it unconditionally at the top level `__init__` -- do we expect majority of datalad imports and invocations use it? probably not... but then it would add overhead (i was optimizing once or twice already to reduce it) for no good reason.  Could it go to e.g. somewhere like datalad/support/ssh.py?